src/test/java/bio/terra/integration/FileTest.java

```diff
@@ -46,7 +46,7 @@
 
 @RunWith(SpringRunner.class)
 @SpringBootTest
-@ActiveProfiles({"google", "integrationtest"})
+@ActiveProfiles({"google", "integrationtest", "alpha"})
```

Comment:
I think this means that the alpha profile will be set to active every time this test is run -- is that what is intended? It seems confusing for when we run this, for example, locally.

Reply:
The alpha profile won't exist locally, so in practice this is a no-op. Unfortunately `@ActiveProfiles` ignores anything from environment variables and `SPRING_PROFILES_ACTIVE`. There may be a way to change the tests to run with modified annotations, but I think that is outside the scope of this PR.

Reply:
Okay sorry for being dense, but then I'm confused what the purpose of the profile is. If it's a no-op and not defined anywhere, then what effect does it have on the test?

Reply:
I think we should not include this and use environment variables for the different test environments.

Reply:
It's defined in the datarepo [alpha.yaml](https://github.com/broadinstitute/terra-helmfile/blob/master/terra/values/datarepo/live/alpha.yaml#L50) .